vaadin-date-time-picker-flow/src/test/java/com/vaadin/flow/component/datetimepicker/DateTimePickerTest.java

```diff
@@ -78,6 +80,24 @@ public void setMaxGetMax() {
         picker.setMax(LocalDateTime.of(2018, 4, 25, 13, 45, 10));
         assertEquals(LocalDateTime.of(2018, 4, 25, 13, 45, 10),
                 picker.getMax());
+
+    @Test
+    public void setI18n() {
+        DateTimePicker picker = new DateTimePicker();
+
+        DatePicker.DatePickerI18n i18n = new DatePicker.DatePickerI18n().setWeek("viikko")
+            .setCalendar("kalenteri").setClear("tyhjennä")
+            .setToday("tänään").setCancel("peruuta").setFirstDayOfWeek(1)
+            .setMonthNames(Arrays.asList("tammiku", "helmikuu", "maaliskuu",
```

Comment:
First string should be "tammikuu" (missing last letter)

Reply:
Updated.